zk/src/eddsa/mod.rs

```diff
@@ -3,20 +3,24 @@ pub mod native;
 
 use crate::{
 	edwards::{
-		params::EdwardsParams, AssignedPoint, IntoAffineChip, PointAddChip, StrictScalarMulChipset,
-		StrictScalarMulConfig,
+		params::{BabyJubJub, EdwardsParams},
+		AssignedPoint, IntoAffineChip, PointAddChip, StrictScalarMulChipset, StrictScalarMulConfig,
 	},
 	gadgets::lt_eq::{LessEqualChipset, LessEqualConfig},
-	params::hasher::RoundParams,
+	params::hasher::{poseidon_bn254_5x5::Params, RoundParams},
 	poseidon::{PoseidonChipset, PoseidonConfig},
 	Chip, Chipset, CommonConfig, FieldExt, RegionCtx,
 };
 use halo2::{
 	circuit::{AssignedCell, Layouter, Region},
+	halo2curves::bn256::Fr as Scalar,
 	plonk::{Error, Selector},
 };
 use std::marker::PhantomData;
 
+/// Type alias for Eddsa chip on BabyJubJub elliptic curve
+pub type Eddsa = EddsaChipset<Scalar, BabyJubJub, Params>;
+
```

Comment:
I think circuits/mod.rs is a better place for it.
We will make circuits also generic over curves and other types and we could have hardcoded definitions in that file for all circuits and chips.

I can change this